integration/grpc_test.go

```diff
@@ -28,6 +29,28 @@ func TestAppendTokenToOutgoingContext(t *testing.T) {
 	}
 }
 
+func ExampleAppendTokenToOutgoingContext() {
+	// someFunc doesn't do anything, but in a real-world situation it might
+	// send a request to some grpc service that requires an authorization
+	// token
+	someFunc := func(context.Context) {
+		// send a request to some grpc service
+	}
+	authToken, err := jwt.NewWithClaims(
+		jwt.SigningMethodHS256, jwt.MapClaims{
+			"user":  "user-test",
+			"roles": "admin",
+		},
+	).SignedString([]byte("some-secret"))
+	if err != nil {
+		log.Fatalf("unable to build token: %v", err)
+	}
+	ctxBearer := AppendTokenToOutgoingContext(
+		context.Background(), authToken, "Bearer",
+	)
+	someFunc(ctxBearer)
+}
```

Comment:
does this test run with `go test`?  I think you'll need an `//Output:` comment in order for it to execute as a test